lib/gui/etcher.ts

```diff
@@ -209,6 +210,7 @@ async function createMainWindow() {
 	);
 
 	const page = mainWindow.webContents;
+	remoteMain.enable(page);
 
```

Comment:
on line 233, `allowRendererProcessReuse`, that property is removed: https://www.electronjs.org/blog/electron-14-0/#removed-appallowrendererprocessreuse

does this remoteMain add that back, or we can delete that line? (same for tests)

Reply:
Yeah, we should remove the flag, I expect it has no effect but for some reason typings allow it to be set which is misleading.